packages.yml

```diff
@@ -2,9 +2,10 @@ packages:
   # - package: fivetran/quickbooks_source
   #   version: [">=0.6.0", "<0.7.0"]
 
+#REMEMBER TO CHANGE REVISION BEFORE MERGING
```

Comment:
Bumping the comment to remember to switch before merging. I actually think since there have been no updates to the source package that this can be updated to reference the latest hub version of the source.

Reply:
After talking with Joe, we'll keep the branch source package since some of the updates are dependent on the newest features which haven't been updated to main.